cluster/gce/trusty/configure-helper.sh

```diff
@@ -537,10 +530,7 @@ start_kube_scheduler() {
   if [ -n "${SCHEDULER_TEST_LOG_LEVEL:-}" ]; then
     log_level="${SCHEDULER_TEST_LOG_LEVEL}"
   fi
-  params="${params} ${log_level}"
-  if [ -n "${SCHEDULER_TEST_ARGS:-}" ]; then
-    params="${params} ${SCHEDULER_TEST_ARGS}"
-  fi
+  params="${params} ${log_level} ${SCHEDULER_TEST_ARGS:-}"
```

Comment:
The block 530--532 for calculating the log level is needed. I think a better way is remove line 528 and make 533 like  params="${log_level} ${SCHEDULER_TEST_ARGS:-}"